src/RNCamera.js

```diff
@@ -411,6 +445,8 @@ export default class Camera extends React.Component<PropsType, StateType> {
           onPictureSaved={this._onPictureSaved}
         >
           {this.renderChildren()}
+          {this.props.rectOfInterest !== undefined &&
```

Comment:
I think drawing rect of interesting should be in user land, instead of here

can you remove it?

Reply:
Done